src/js/components/MaskedInput/doc.js

```diff
@@ -84,6 +84,12 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '18px',
   },
+  'maskedInput.disabled.opacity': {
+    description: 'The opacity when the MaskedInput is disabled.',
+    type: 'number',
```

Comment:
I'm thinking boolean and string are also acceptable values, can you confirm?

Reply:
I will test it out and update the docs if needed.

Reply:
great, opacity should be able to accept t-shirt sizes ("small",...)

Reply:
String is also accepted, however applying a boolean did not have the intended effect.